gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
+#Mon Oct 07 10:26:18 CEST 2024
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

Comment:
Question: Are there any dependencies incompatible with gradle 8.4?

Reply:
Not sure, this seems to might have been updated automatically after using gradle commands, would there be a problem with using the most modern version anyway?

Reply:
We will keep this line, but in case of a Gradle build problem, this will be reverted.